crates/swc_common/src/syntax_pos.rs

```diff
@@ -385,71 +385,71 @@ extern "C" {
 
 impl Span {
     #[inline]
-    pub fn lo(self) -> BytePos {
+    pub const fn lo(self) -> BytePos {
         self.lo
     }
 
     #[inline]
-    pub fn new(mut lo: BytePos, mut hi: BytePos) -> Self {
-        if lo > hi {
-            std::mem::swap(&mut lo, &mut hi);
+    pub const fn new(lo: BytePos, hi: BytePos) -> Self {
+        if lo.0 > hi.0 {
```

Comment:
This should be removed and callers should be fmodified to call Span::new with correct argument